README.md

```diff
@@ -6,17 +6,23 @@ automate a headless Chrome browser](features/support/env.rb).
 
 Smokey is run automatically:
 
-- every few minutes via a
-  [cronjob](https://argo.eks.integration.govuk.digital/applications/cluster-services/smokey)
+- every few minutes via a [cronjob][smokey-argo]
 - by the [deployment
   automation](https://argo-workflows.eks.integration.govuk.digital/workflows?limit=100)
   to determine whether to promote a release to the next environment (staging,
   production)
 
+Smokey can also be run manually by visiting the [Smokey project in Argo][smokey-argo],
+clicking the three dots against the 'smokey' cronjob and then
+selecting 'Create Job'. Within a couple of minutes, this will create a
+new 'smokey' run.
```

Comment:
Best to avoid unnecessary use of passive voice etc.
```suggestion
You can also start a Smokey run [from Argo CD][smokey-argo]. Click the three
dots in the 'smokey' cronjob and choose 'Create Job'. It can take a few seconds
for the job to appear.
```